recipes/python-pptx/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{%set name = "python-pptx" %}
+{%set version = "0.5.8" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "de591ece301d750cc22e8154716a6d6b643cd99e82451b5cb3e799c8e07d6a4d" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - lxml >=3.1.0
+    - pillow >=2.6.1
+    - xlsxwriter >=0.5.7
+
+test:
+  imports:
+    - pptx
+    - pptx.chart
+    - pptx.compat
+    - pptx.dml
+    - pptx.enum
+    - pptx.opc
+    - pptx.oxml
+    - pptx.oxml.chart
+    - pptx.oxml.dml
+    - pptx.oxml.parts
+    - pptx.oxml.shapes
+    - pptx.parts
+    - pptx.shapes
+    - pptx.text
+
+about:
+  home: http://github.com/scanny/python-pptx
+  license: MIT
```

Comment:
Placed in issue ( https://github.com/conda-forge/python-pptx-feedstock/issues/1 ).